example/config.ru

```diff
@@ -1,5 +1,6 @@
 require 'bundler/setup'
 require 'sinatra/base'
+require 'active_support/core_ext/hash'
```

Comment:
Why was this change needed?

Reply:
because it raises an error `undefined method 'with_indifferent_access' for #<Hash:0x1012392c0>` I don't see any other filed(except integration_test.rb) `active_support/core_ext/hash` has been required.